TodoApp/src/TodoApp.Contracts/Models/RetrievedItem.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace TodoApp.Contracts.Models
+{
+    public class RetrievedItem
+    {
+        private Item _item;
+
+        public Item Item
+        {
+            get => WasFound
+                ? _item
+                : throw new InvalidOperationException();
```

Comment:
Please, add a message to the exception's constructor that would lead a developer to the `WasFound` property.